examples/dropwizard/README.md

```diff
@@ -0,0 +1,40 @@
+# Containerize a [Dropwizard](https://dropwizard.io) application with Jib
+
+## How to start the Dropwizard application
+
+1. Run `./mvnw clean package -Djib.to.image=dropwizard-jib-example` to build your container
+1. Start the application
+    - **Without Docker**: `./mvnw exec:java`
+    - **With Docker**: `docker run --rm -p 8080:8080 dropwizard-jib-example:1`
+1. Check that your application is running at http://localhost:8080/hello-world
+
+## Health Check
+
+See your application's health at http://localhost:8080/admin/healthcheck
+
+## Extras
+
+Freemaker templating is setup for [`config.yml`](src/main/jib/app/config/dropwizard.yml) through [`tkrille/dropwizard-template-config`](https://github.com/tkrille/dropwizard-template-config); this allows one to heavily customize the properties file via the container environment with FTL conditional checks and for loops, for example.
```

Comment:
Freemaker -->
Freemarker

Maybe FreeMarker.

`config.yml` --> `dropwizard.yml`

Reply:
Whoops. 

Hmm. IntelliJ caught the path rename, just not the link text.